web/index.ts

```diff
@@ -334,7 +358,9 @@ const App = (_: any, state: AppState, setState: SetState) => {
   url.pathname = `${encodeURIComponent(text)}.${fileType}`
   url.searchParams.append('theme', theme)
   url.searchParams.append('md', mdValue)
+  url.searchParams.append('bgtype', backgroundType)
```

Comment:
```suggestion
  url.searchParams.append('backgroundType', backgroundType)
```

Could you rename the url param to `backgroundType`? I don't care what the name is but would like it to be consistent. Right now we have `bgtype` and `backgroundType` in various places.